File: StockManagement.Entities/Concrete/Warehouse.cs
﻿using StockManagement.Core.Entities;
using System.Collections.Generic;

namespace StockManagement.Entities.Concrete
{
    public class Warehouse : BaseEntity, IEntity
    {
        public string WarehouseName { get; set; }

        public string Address { get; set; }

        public int CityId { get; set; }
        public City City { get; set; }
        public virtual ICollection<Product> Products { get; set; }
    }
}

File: StockManagement.UI/src/app/models/Warehouse/Warehouse.ts
import { City } from '../City/City';

export class Warehouse {
    id:number;
    warehouseName :string;
    address:string;
    cityid:number;
    city : City;
    createDate:Date;
    createdBy:string;
    modifiedBy:string;
    modifiedDate:Date;

}
